velox/exec/OperatorUtils.h

```diff
@@ -158,4 +158,11 @@ void projectChildren(
     int32_t size,
     const BufferPtr& mapping);
 
+/// Merge the sourve row vector to dest row vector.
+void mergeVector(
```

Comment:
Can you use BaseVector::append method instead?

Reply:
BaseVector::append will copy source vector to targetIndex which is previousSize=BaseVector::size(), so bufferInputs_ initial size should be 0, the resize in BaseVector::append may cause vector copy because previous capacity is smaller than new size, The benchmark show that when use BaseVector::append, performance will decrease.
```
  if (!bufferInputs_) {
    bufferInputs_ = BaseVector::create<RowVector>(
        vector->type(), preferredOutputBatchRows_, pool_);
  }
  mergeVector(bufferInputs_, vector, bufferRows_, numInput);
```
The code change to:
```
  if (!bufferInputs_) {
    bufferInputs_ = BaseVector::create<RowVector>(
        vector->type(), preferredOutputBatchRows_, pool_);
    bufferInputs_->resize(0);
  }
  bufferInputs_->append(vector.get());
```

Reply:
> the resize in BaseVector::append may cause vector copy because previous capacity is smaller than new size, 

Given that you first create vector of size preferredOutputBatchRows_, then call resize(0), I don't expect resize in append to need to make a copy. Is this happening? Maybe that's a bug that needs fixing? 

> The benchmark show that when use BaseVector::append, performance will decrease.

How much overall performance decreases? Can you look into why append is less performance than custom 'mergeVector'?